Update Team and Business onboarding (#16249)
docker/docs: docker-hub/onboard-business.md

1. Go to [Billing Details](https://hub.docker.com/billing){: target="_blank" rel="noopener" class="_"} and then select on your organization's name.
2. Under **Plan**, view your subscription. If you organization has a Docker Business subscription, you will see **Docker Business**.

## Step 3: Invite owners

Now that you have a Docker Business organization, it's time to start adding owners to help you set up and manage your organization. Owners can add or remove members, and configure Single Sign-on as well as other security settings.

To add an owner, invite a user to the **owners** team. For more details, see [Invite members](../docker-hub/members.md/#invite-members){: target="_blank" rel="noopener" class="_"}.

## Step 4: Invite members

Add members to your organization using a CSV file, or by entering their email addresses. For more details, see [Invite members](../docker-hub/members.md/#invite-members){: target="_blank" rel="noopener" class="_"}

## Step 5: Manage members with SSO and SCIM

Automate adding members to your organization using the following:

>    - Single Sign-on: Automatically provision and add members when they sign in to Docker through your identity provider. For details, see [Single Sign-on overview](../single-sign-on/index.md).

Reviewer: Should we say Docker Hub instead of Docker?

Author: Thanks. Updated in the commit below.

- [Consolidate a repository](../docker-hub/repos/index.md/#consolidating-a-repository) from your personal account to your organization.
- Create [teams](../docker-hub/orgs.md/#create-a-team) and configure [repository permissions](../docker-hub/orgs.md/#configure-repository-permissions).
- Configure [Hardened Docker Desktop](../desktop/hardened-desktop/index.md) to improve your organization’s security posture for containerized development. Hardened Docker Desktop includes:
   - [Settings Management](../desktop/hardened-desktop/settings-management/index.md), which helps you to confidently manage and control the usage of Docker Desktop within your organization.
   - [Enhanced Container Isolation](../desktop/hardened-desktop/enhanced-container-isolation/index.md), a setting that instantly enhances security by preventing containers from running as root in Docker Desktop’s Linux VM.

>    - [Image Access Management](../docker-hub/image-access-management.md/), which gives you control over which images developers can pull from Docker Hub.

Reviewer: ```suggestion
   - [Image Access Management](../docker-hub/image-access-management.md/), lets you control which images developers can pull from Docker Hub.
```

>    - [Registry Access Management](../docker-hub/registry-access-management.md/), which allows you to control the registries developers can access.

Reviewer: ```suggestion
   - [Registry Access Management](../docker-hub/registry-access-management.md/), lets you control the registries developers can access.
```
